sympy/functions/special/error_functions.py

```diff
@@ -1895,6 +1907,19 @@ def _eval_rewrite_as_sinc(self, z, **kwargs):
         t = Symbol('t', Dummy=True)
         return Integral(sinc(t), (t, 0, z))
 
+    def _eval_as_leading_term(self, x, logx=None, cdir=0):
+        arg = self.args[0].as_leading_term(x)
+        arg0 = arg.subs(x, 0)
+
+        if arg0 is S.NaN:
+            arg0 = arg.limit(x, 0, dir='-' if re(cdir).is_negative else '+')
```

Comment:
some thoughts over the code:
`arg0 = arg.limit(x, 0, dir='-' if re(cdir).is_negative else '+')`
maybe it's better to move dir calculation to temp variable to make it more readable